resolwe_bio/utils/test.py

```diff
@@ -63,10 +68,31 @@ class BioProcessTestCase(ProcessTestCase):
     """
 
     def setUp(self):
-        """Initialize test files path."""
+        """Initialize test files path and species annotation."""
         super(BioProcessTestCase, self).setUp()
         self.files_path = TEST_FILES_DIR
 
+        general_group = AnnotationGroup.objects.create(name="general", sort_order=1)
+        AnnotationField.objects.create(
+            name="species",
+            sort_order=1,
+            group=general_group,
+            type=AnnotationType.STRING.value,
+            vocabulary={
+                "Caenorhabditis elegans": "Caenorhabditis elegans",
+                "Cricetulus griseus": "Cricetulus griseus",
+                "Dictyostelium discoideum": "Dictyostelium discoideum",
+                "Dictyostelium purpureum": "Dictyostelium purpureum",
+                "Drosophila melanogaster": "Drosophila melanogaster",
+                "Homo sapiens": "Homo sapiens",
+                "Macaca mulatta": "Macaca mulatta",
+                "Mus musculus": "Mus musculus",
+                "Odocoileus virginianus texanus": "Odocoileus virginianus texanus",
```

Comment:
Some of the values should not be here. I guess it auto-completed the list. Will fix the vocabulary after the tests pass.